file-formats/ints/type/zif_aff_ints_v1.intf.abap

```diff
@@ -0,0 +1,390 @@
+      tkey_auto_switch_on   TYPE abap_bool,
+      "! <p class="shorttext">Turnkey Implementation Type</p>
+      "! Turnkey Implementation Type
+      "! $required
+      tkey_type             TYPE c LENGTH 10,
+      "! <p class="shorttext">Scenario Technology</p>
+      "! Scenario Technology
+      scenario_tech         TYPE c LENGTH 20,
+      "! <p class="shorttext">Prediction class</p>
+      "! Prediction class
+      object_name           TYPE c LENGTH 30,
+      "! <p class="shorttext">Prerequisite Check Class</p>
+      "! Prerequisite Check Class
+      prereq_chk_class      TYPE c LENGTH 30,
+      "! <p class="shorttext">Scenario Technical Id</p>
+      "! Scenario Technical Id
+      scenario_tech_id      TYPE c LENGTH 64,
+      "! <p class="shorttext">State of Intelligent Scenario</p>
+      "! State of Intelligent Scenario
+      state                 TYPE c LENGTH 10,
+      "! <p class="shorttext">ABAP Language Version</p>
+      "! ABAP Language Version
+      abap_language_version TYPE c LENGTH 1,
+      "! <p class="shorttext">ISLM Framework Version</p>
+      "! ISLM Framework Version
+      ism_version           TYPE p LENGTH 4 DECIMALS 2,
+      "! <p class="shorttext">Turnkey Class</p>
+      "! Turnkey Class
+      tkey_class            TYPE c LENGTH 30,
+      "! <p class="shorttext">Intelligent Scenario Usage Type</p>
+      "! Intelligent Scenario Usage Type
+      usage_type            TYPE c LENGTH 10,
+      "! <p class="shorttext">OAuth 2.0 Client Profile</p>
+      "! OAuth 2.0 Client Profile
+      oauth_profile         TYPE c LENGTH 30,
+    END OF ty_islm_ps,
+
+    "! <p class="shorttext">Scenario Binding Mappings</p>
+    "! Scenario Binding Mappings
+    BEGIN OF ty_islm_psbindmp,
+      "! <p class="shorttext">Signature Name</p>
+      "! Signature Name
+      signature_name TYPE c LENGTH 30,
+      "! <p class="shorttext">Table or view field name</p>
+      "! Table or view field name
+      from_field     TYPE c LENGTH 30,
+      "! <p class="shorttext">Table or view field name</p>
+      "! Table or view field name
+      to_field       TYPE c LENGTH 30,
+    END OF ty_islm_psbindmp,
+
+    "! <p class="shorttext">Dataset Bindings</p>
+    "! Dataset Bindings
+    BEGIN OF ty_islm_psbind,
+      "! <p class="shorttext">Name</p>
+      "! Name
+      name      TYPE c LENGTH 30,
+      "! <p class="shorttext">Reference UUID</p>
+      "! Reference UUID
+      reference TYPE c LENGTH 36,
+    END OF ty_islm_psbind,
+
+    "! <p class="shorttext">Scenario Signatures</p>
+    "! Scenario Signatures
+    BEGIN OF ty_islm_pssig,
+      "! <p class="shorttext">Signature Name</p>
+      "! Signature Name
+      name           TYPE c LENGTH 30,
+      "! <p class="shorttext">Signature Type</p>
+      "! Signature Type
+      signature_type TYPE c LENGTH 32,
+      "! <p class="shorttext">Description</p>
+      "! Description
+      description    TYPE c LENGTH 100,
```

Comment:
Regarding this description, is this info coming from another object type? I am asking this to avoid redundancy, as information that is present in other sources do not belong in this AFF

Reply:
This is the description of Signature which belongs to INTS object only. This table is a part of INTS in SOBJ.